bundles/net.tourbook/src/net/tourbook/importdata/RawDataManager.java

```diff
@@ -1504,6 +1538,239 @@ public void clearInvalidFilesList() {
+       * imported
+       */
+      final TourData oldTourDataInImportView = _toursInImportView.remove(tourId);
+
+      TourData clonedTourData = null;
+      try {
+
+         clonedTourData = (TourData) tourData.clone();
+
+         // loop: For each tour value type, we save the associated data for future display
+         //to compare with the new data
+         for (final TourValueType tourValueType : tourValueTypes) {
+
+            switch (tourValueType) {
+
+            case TIME_SLICES_TIME:
+               for (int index = 0; index < tourData.timeSerie.length; ++index) {
+                  tourData.timeSerie[index] = 0;
+               }
+               tourData.setTourDeviceTime_Elapsed(0);
+               tourData.setPausedTime_Start(null);
+               tourData.setPausedTime_End(null);
+               tourData.setTourDeviceTime_Paused(0);
+               tourData.setTourDeviceTime_Recorded(0);
+               tourData.setTourComputedTime_Moving(0);
+               tourData.clearComputedSeries();
+               break;
+            case TOUR_MARKER:
+               clonedTourData.setTourMarkers(new HashSet<>(tourData.getTourMarkers()));
+
+               tourData.setTourMarkers(new HashSet<>());
+               break;
+
+            case TIME_SLICES_CADENCE:
+               clonedTourData.setCadenceSerie(tourData.getCadenceSerie());
+               clonedTourData.setAvgCadence(tourData.getAvgCadence());
+               clonedTourData.setCadenceMultiplier(tourData.getCadenceMultiplier());
+
+               tourData.setCadenceSerie(null);
+               tourData.setAvgCadence(0);
+               tourData.setCadenceMultiplier(0);
+               tourData.setCadenceZone_SlowTime(0);
+               tourData.setCadenceZone_FastTime(0);
+               break;
+
+            case TIME_SLICES_ELEVATION:
+               clonedTourData.setTourAltDown(tourData.getTourAltDown());
+               clonedTourData.setTourAltUp(tourData.getTourAltUp());
+
+               tourData.altitudeSerie = null;
+               tourData.setTourAltDown(0);
+               tourData.setTourAltUp(0);
+               break;
+
+            case TIME_SLICES_GEAR:
+               clonedTourData.setFrontShiftCount(tourData.getFrontShiftCount());
+               clonedTourData.setRearShiftCount(tourData.getRearShiftCount());
+
+               tourData.setFrontShiftCount(0);
+               tourData.setRearShiftCount(0);
+               break;
+
+            case TIME_SLICES_POWER_AND_PULSE:
+               clonedTourData.setPower_Avg(tourData.getPower_Avg());
+               clonedTourData.setAvgPulse(tourData.getAvgPulse());
+               clonedTourData.setCalories(tourData.getCalories());
+
+               tourData.setPowerSerie(null);
+               tourData.setPower_Avg(0);
+               tourData.pulseSerie = null;
+               tourData.setAvgPulse(0);
+               tourData.setCalories(0);
```

Comment:
I replicated the same that was done all those years for [re-import](https://github.com/wolfgang-ch/mytourbook/blob/df98402e4bb417c74301276a260e92439911ea94/bundles/net.tourbook/src/net/tourbook/importdata/RawDataManager.java#L1078)

```
 case TIME_SLICES_POWER_AND_PULSE:
                  clonedTourData.setPower_Avg(oldTourData.getPower_Avg());
                  clonedTourData.setAvgPulse(oldTourData.getAvgPulse());
                  clonedTourData.setCalories(oldTourData.getCalories());
                  break;

               case TIME_SLICES_POWER_AND_SPEED:
                  clonedTourData.setPower_Avg(oldTourData.getPower_Avg());
                  clonedTourData.setCalories(oldTourData.getCalories());
                  break;
```

Reply:
`setCalories()` was added 2016 in 564b815aa3b7825412e7b17bac00ba807ec6036e maybe I thought it's energy or I was too lazy to create a new property in the UI,...

Reply:
Do you want me to separate them or to keep them as is ?

Reply:
I think it is better to separate them but then you have to do it for re-import and delete action. It would be great if you can do it, I just extended my todo list in the readme file

Reply:
I added it in the readme file. I will do it in a next PR.